scipy/stats/tests/test_distributions.py

```diff
@@ -312,6 +312,78 @@ def test_pmf(self):
         assert_equal(val, 0)
 
 
+class TestGenInvGauss(object):
+    def setup_method(self):
+        np.random.seed(1234)
+
+    @pytest.mark.slow
+    def test_rvs_with_mode_shift(self):
+        # ratio_unif w/ mode shift
+        gig = stats.geninvgauss(2.3, 1.5)
+        _, p = stats.kstest(gig.rvs(size=1500, random_state=1234), gig.cdf)
+        assert_equal(p > 0.05, True)
+
+    @pytest.mark.slow
+    def test_rvs_without_mode_shift(self):
+        # ratio_unif w/o mode shift
+        gig = stats.geninvgauss(0.9, 1.5)
```

Comment:
This test case needs to be adjusted: current parameters use mode shift (since `b > 1`). I will wait for further feedback before changing it